Code-the-Dream-School/ffprac-team2-back
# PRAC - 14 Tutor schema  implementation along with routers and controllers to handle CRUD operations.

## One Line Description
Tutor schema  implementation along with routers and controllers to handle CRUD operations.
## Requirements
Build a Tutor model to represent tutor data.
Implement routers and controllers for CRUD operations related to tutors.

## Test Steps
Retrieve all tutors: Send a GET request to the /tutors endpoint. 
Create a tutor: Send a POST request to the /tutors endpoint with valid tutor data. Verify that the tutor is successfully created and stored in the database.
Update a tutor: Send a PATCH request to the /tutors/{tutorId} endpoint with updated tutor information. Verify that the tutor data is correctly updated in the database.
Delete a tutor: Send a DELETE request to the /tutors/{tutorId} endpoint with a valid tutor ID. Confirm that the tutor is removed from the system.
_Instructions on how to test the changes_
